src/models/connectionProfile.ts

```diff
@@ -107,7 +108,8 @@ export class ConnectionProfile extends ConnectionCredentials implements IConnect
 				name: LocalizedConstants.tenant,
 				message: LocalizedConstants.azureChooseTenant,
 				choices: tenantChoices,
-				shouldPrompt: (answers) => profile.isAzureActiveDirectory() && tenantChoices.length > 1,
+				default: defaultProfileValues ? defaultProfileValues.tenantId : undefined,
+				shouldPrompt: (answers) => profile.isAzureActiveDirectory() && tenantChoices.length > 1 && !getEnableSqlAuthenticationProviderConfig(),
```

Comment:
Would suggest adding a comment here explaining why this isn't necessary when SQL auth provider is enabled